스프린트_미션4/signup.js

```diff
@@ -0,0 +1,107 @@
+document.addEventListener('DOMContentLoaded', function() {
+  const email = document.querySelector('#email');
+  const email_error = document.querySelector('#email_error');
+  const nickname = document.querySelector('#nickname');
+  const nickname_error = document.querySelector('#nickname_error');
+  const password = document.querySelector('#password');
+  const password_error = document.querySelector('#password_error');
+  const password_check = document.querySelector('#passwordConfirmation');
+  const password_check_error = document.querySelector('#password_check_error');
+  const signUpForm = document.querySelector('form');
+  const signUpButton = document.querySelector('#signup-button');
+
+
+     // 버튼 활성화 비활성화 설정
+     function signUpButtonAbled() {
+      if (
+        email.value.trim() !== '' &&
+        nickname.value.trim() !== '' &&
+        password.value.trim() !== '' &&
+        password_check.value.trim() !== '' &&
+        !email.classList.contains('error-color') &&
+        !nickname.classList.contains('error-color') &&
+        !password.classList.contains('error-color') &&
+        !password_check.classList.contains('error-color')
+      ) {
+        signUpButton.disabled = false;
+      } else {
+        signUpButton.disabled = true;
+      }
+    }
+  
+    signUpForm.addEventListener('submit', function(event) {
+      event.preventDefault();
+      
+      // 회원가입 페이지 함수
+      changeToSignUp();
+    });
+  
+    function changeToSignUp() {
+      window.location.href = '/signup';

```

Comment:
주요 변경 사항에 올려주신 질문에 대한 해답으로 단순히 signup으로 가서는 안되고 signup.html을 붙여주셔야합니다~
```suggestion
      window.location.href = '/signup.html';
```

만일 signup 으로 가고싶으시다면
signup 디렉토리를 만든 후 index.html 파일을 만들어주시면 됩니다!